util/launcher/src/migrations/rebuild_block_filter.rs

```diff
@@ -0,0 +1,29 @@
+use ckb_db::{Result, RocksDB};
+use ckb_db_migration::{Migration, ProgressBar};
+use ckb_db_schema::{COLUMN_META, META_LATEST_BUILT_FILTER_DATA_KEY};
+use std::sync::Arc;
+
+pub struct RebuildBlockFilter;
+
+const VERSION: &str = "20220803143233";
```

Comment:
If I understand correctly, this file can be squashed (removed) later, since we haven't release any official `CKB` binaries.

Reply:
Yes, I've add a code comment